test/json/artists.json

```diff
@@ -31,7 +31,7 @@
                "year":1889
             },
             {
-               'name':"Starry Night Over the Rhone",
+               "name":"Starry Night Over the Rhone",
```

Comment:
This one also needs to reverted.  Single quotes are part of the test.  I know we can improve on this quite a bit. I am open to PRs :D